ci/build.yml

```diff
@@ -250,6 +250,9 @@ jobs:
   dependsOn:
     - da_ghc_lib
     - check_for_release
+  condition: and(succeeded(),
+                 not(and(eq(dependencies.check_for_release.outputs['out.is_release'], 'true'),
+                         eq(variables['Build.SourceBranchName'], 'main'))))
```

Comment:
do we want to run them on PRs? I guess there is no harm but it also doesn’t seem super useful

Reply:
Fair enough.